app/models/container_project.rb

```diff
@@ -69,4 +70,43 @@ def disconnect_inv
     self.deleted_on = Time.now.utc
     save
   end
+
+  def add_role_to_user(user_name, role_name)
```

Comment:
@kevensen this is not just adding a role IIUC, it is also changing an existing role if exists, so the name is misleading.